tests/test_clone.py

```diff
@@ -116,3 +117,75 @@ def test_convert_1D_backward_slope(slope, helpers):
             assert layer.slope == Slope(slope)
         if layer_class_name in ("BackwardDense", "BackwardConv2D"):
             assert layer.slope == Slope(slope)
+
+
+# test different convex domains
+@pytest.mark.parametrize(
+    "p",
+    [2, np.inf],
+)
+def test_convert_1D(p, n, method, mode, floatx, helpers):
```

Comment:
Beware, this function is already defined in the same file. You should modify the name (something like test_convert_1D_ball ?)